Artifacts/MsSql/Structure/Admin/0161-Rename-OdsInstancederivative.sql

```diff
@@ -0,0 +1,42 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE [dbo].[OdsInstanceDerivative]
+DROP CONSTRAINT [FK_OdsInstanceDerivative_OdsInstanceId_OdsInstanceId];
+
+ALTER TABLE [dbo].[OdsInstanceDerivative]
+DROP CONSTRAINT [UC_OdsInstanceDerivative_OdsInstanceId_DerivativeType];
```

Comment:
Before trying to drop Constraint, Please check Exists or not , Then if exists drop Constraint